src/services/mentees.js

```diff
@@ -1011,12 +1019,19 @@ module.exports = class MenteesHelper {
 			}
 
 			const query = utils.processQueryParametersWithExclusions(queryParams)
+			const userExtensionModelName = await menteeQueries.getModelName()
+			const mentorExtensionModelName = await menteeQueries.getModelName()
 
 			let validationData = await entityTypeQueries.findAllEntityTypesAndEntities({
 				status: common.ACTIVE_STATUS,
+				model_names: { [Op.overlap]: [userExtensionModelName, mentorExtensionModelName] },
```

Comment:
I think we should use overlap only when necessary as given here. Other instances can be without overlap, instead contains should suffice.